src/vstd_ext/set_lib.rs

```diff
@@ -66,4 +78,13 @@ proof fn element_in_seq_exists_in_original_finite_set<A>(s: Set<A>, e: A)
     }
 }
 
-}
+pub proof fn antisymmetry_of_set_equality<A>(s1: Set<A>, s2: Set<A>)
```

Comment:
Do we really need this one?

Reply:
@Catoverflow Do we really need this lemma? Can we replace the usage of this lemma with an assertion on the extensional equality?